zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -415,7 +424,8 @@ def make_tasks_with_intermediaries(  # pylint: disable=too-many-locals
         op_kwargs: P.kwargs,
     ) -> Tuple[List[mazepa.tasks.Task[R_co]], VolumetricBasedLayerProtocol | None]:
         dst_temp = self._get_temp_dst(dst, idx, self.flow_id)
-        idx_chunks = self.processing_chunker(idx, mode="exact")
+        have_gap = self.gap is not None and self.gap != Vec3D[int](0, 0, 0)
+        idx_chunks = self.processing_chunker(idx, mode="expand" if have_gap else "exact")
```

Comment:
What happens if it's `exact`? If the arguments are correct, then this should never need to be `expand`. Can you check the `idx_chunks` when it's `exact` and compare them to `expand`?

Reply:
If it's `exact` then code further on (in bbox_strider.py) throws this exception:

```
NotImplementedError: `exact` mode is only supported when the `chunk_size` and `stride` are equal
```

And of course they're _not_ equal here; the whole point of this is to specify a stride bigger than the chunk, so it leaves gaps.

Are you suggesting that we should update bbox_strider so it can deal with this case?

Reply:
I'll make an issue.